### ISSUE TITLE
8.5.1 stockticker example outdated

### ISSUE THREAD

**Issue by yang-guangyuan** [espertechinc/nesper#28]
com.espertech.esper.common.client.EPRuntimeException:“com.espertech.esper.common.client.EPRuntimeException: Error during compilation: Incorrect syntax near 'every' (a reserved keyword) [every Pricelimit=PriceLimit()] [every Pricelimit=PriceLimit()]”

I do not know why, who can help me  thanks!

**Comment by yang-guangyuan** [comment]
// Listen to all limits to be set
            var expressionText = "every Pricelimit=PriceLimit()";
            EPStatement factory = runtime.DeployStatement(expressionText);

**Comment by bernhardttom** [comment]
"select * from PriceLimit" is what you want

**Comment by yang-guangyuan** [comment]
I use examples code, but compile  run  is  error  
Error during compilation: Incorrect syntax near 'every' (a reserved keyword) [every Pricelimit=PriceLimit()] [every Pricelimit=PriceLimit()]

**Comment by bernhardttom** [comment]
what example?

**Comment by yang-guangyuan** [comment]
NEsper.Examples.StockTicker

nesper-release_8.5.1\examples\NEsper.Examples\example\stockticker

thanks

**Comment by yang-guangyuan** [comment]
NEsper.Examples.StockTicker

nesper-release_8.5.1\examples\NEsper.Examples\example\stockticker

thanks

**Comment by yang-guangyuan** [comment]
@bernhardttom thanks

**Comment by ajaxx** [comment]
I'll take a look at that example Tom. I'm doing testing in this area anyway.

**Comment by yang-guangyuan** [comment]
![image](https://user-images.githubusercontent.com/80253379/124211913-8616a700-db20-11eb-97d2-29c97f474d91.png)
In 8.5.2

**Comment by ajaxx** [comment]
Yes, I was fixing this as part of the 8.6.0 work.  Is this something you need working as part of 8.5.2 or can it wait until 8.6.0.  If it is immediate and you can fix it, then please submit a PR and I will review it.
